django_prices_vatlayer/__init__.py

```diff
@@ -0,0 +1,21 @@
+from prices import Tax
+
+from .utils import get_tax_for_country
+
+from django.core.exceptions import ObjectDoesNotExist
+
+
+class EuropeanVAT(Tax):
```

Comment:
Please implement `__repr__`  to deliver a meaningful name to `price.history` object